src/components/Page/index.tsx

```diff
@@ -1,5 +1,6 @@
+import React from 'react'
 import { PagePanel } from './styled'
 
-export default ({ children, style }: { children: any; style?: object }) => (
+export default ({ children, style }: { children: React.ReactNode; style?: React.CSSProperties }) => (
```

Comment:
`React.ReactNode` could be replaced by `React.ReactChild`

Reply:
Got it, I also found that `React.ReactChild` has been marked `deprecated` at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L225